src/ConnectedMode/Migration/MigrationChecker.cs

```diff
@@ -70,7 +71,7 @@ private void DisplayMigrationPromptIfMigrationIsNeeded()
             if (obsoleteConfigurationProvider.GetConfiguration()?.Mode != SonarLintMode.Standalone
                 && configurationProvider.GetConfiguration()?.Mode == SonarLintMode.Standalone)
             {
-                migrationPrompt.Show();
+               migrationPrompt.ShowAsync().Forget();
```

Comment:
We wanted to do this with .Forget correct?

Reply:
Yes, I think so. It's simplest for now.